server/plugins/AntiCheatReloaded/checks.yml

```
@@ -82,7 +82,7 @@ speed:
   # Debugs as 'moved too fast on ground'
   groundSpeed:
     # Should this part of the check be enabled?
-    enabled: true
+    enabled: false
     # Initial speed limit value
     initialLimit: 0.34
```

idk what's this but i think this needs a higher value
```suggestion
    initialLimit: 0.50
```